waterbox/bsnescore/bsnes/sfc/ppu-fast/ppu.hpp

```diff
@@ -360,13 +361,23 @@ struct PPU : PPUcounter {
     uint height = 0;
   } frame;
 
-  struct Mode7LineGroups {
+  /*struct Mode7LineGroups {
     int count = -1;
     int startLine[32];
     int endLine[32];
     int startLerpLine[32];
     int endLerpLine[32];
-  } mode7LineGroups;
+  } mode7LineGroups;*/
+
+  typedef struct {
+    int count = -1;
+    int startLine[32];
+    int endLine[32];
+    int startLerpLine[32];
+    int endLerpLine[32];
+  } Mode7LineGroups;
+  
+  Mode7LineGroups* mode7LineGroups;
```

Comment:
Am I seeing it right that this is saving exactly 516 bytes? If so, I'm against having to do such a big change/diff like in `mode7hd.cpp` just to save this little space (it's like 1 or 2 promille of the entire savestate size, not worth it).